opnode/l2/api.go

```diff
@@ -2,9 +2,13 @@
 package l2
 
 import (
+	"bytes"
 	"fmt"
+	"math/big"
 	"reflect"
 
+	"github.com/ethereum/go-ethereum/trie"
```

Comment:
We should probably be using `goimports` or something, but your imports here are split from the rest of the `go-ethereum` imports

Reply:
I mostly just let the IDE handle imports, and run `golangci-lint run -E asciicheck,goimports,misspell --fix`. Will clean up imports, but we should update the `linting.md` doc if it needs to be different.